2.4.0.M2/arm64/alpine/Dockerfile

```diff
@@ -0,0 +1,93 @@
+# openhab image
+#
+# ------------------------------------------------------------------------------
+#               NOTE: THIS DOCKERFILE IS GENERATED VIA "update.sh"
+#
+#                       PLEASE DO NOT EDIT IT DIRECTLY.
+# ------------------------------------------------------------------------------
+#
+FROM multiarch/alpine:arm64-v3.7
```

Comment:
Looks like you are missing some commits in your branch because a PR was merged to upgrade to alpine 3.8.

Reply:
OK, I update to 3.8